src/Internal/Tools/Hashdict.elm

```diff
@@ -151,6 +151,14 @@ insert v (Hashdict h) =
     Hashdict { h | values = Dict.insert (h.hash v) v h.values }
 
 
+{-| Since the Hashdict contains a hash function, the == operator does not work
+simply. Instead, you should use the isEqual operator.
+-}
+isEqual : Hashdict a -> Hashdict a -> Bool
+isEqual h1 h2 =
+    toList h1 == toList h2
```

Comment:
If value `a` does not support the `(==)` operator, this will cause a runtime error. This should be fixed, or it should be made clear that this function is exclusively for `elm-test` to be used.